ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2318,6 +2318,12 @@ sflow:
 
 ```yaml
 snmp_server:
+  engine_ids:
+    local: < engine_name in hex >
+    remotes:
+      - id: < engine_name in hex >
+        address: <  hostname_or_ip_of_remote_engine >
+        port: < udp_port >
```

Comment:
```suggestion
        udp_port: < integer >
```

Reply:
so happy to change this but this is not aligned with the Vxlan udp_port description in the same document: `udp_port: < udp_port >` nor the cvaddrs suggested formatting: `< ip/fqdn >:<port>` or the vrf logging ports: `ports: - < custom_port_1 >`. Is there a rule regarding what the description should be?

Reply:
I vote for `udp_port: < udp_port >`